src/wattsi.pas

```diff
@@ -1091,6 +1091,19 @@    TCrossReferences = record
    const
       CommitSnapshotBaseURL: AnsiString = '/commit-snapshots/';
       SourceGitBaseURL: AnsiString = 'https://github.com/whatwg/html/commit/';
+      kExport = 'export';
+      kDataExport = 'data-export';
+      kFor = 'for';
+      kDataDFNFor = 'data-dfn-for';
+      kDataDFNType = 'data-dfn-type';
+      // NOTE: The following array has the subset of dfn types we actually use
+      // in HTML currently. The full set of ~40 supported dfn types is at
+      // https://github.com/tabatkins/bikeshed/blob/master/bikeshed/config/dfnTypes.py#L7
+      // So if we ever start using any types other than the following in HTML,
+      // then at that time we will also need to update this array.
```

Comment:
We definitely use more than these in HTML, but perhaps not in the PR. For example most of the IDL types would make sense. I'd at least like to have attr-value, element-state, constructor, callback, dictionary, dict-member, enum, typedef.

Reply:
> We definitely use more than these in HTML, but perhaps not in the PR.

ah, OK, yeah — I was just basing it on what we needed for that PR

> For example most of the IDL types would make sense. I'd at least like to have attr-value, element-state, constructor, callback, dictionary, dict-member, enum, typedef.

I guess I can just go ahead and add the complete set of ~40 dfn types that Bikeshed code shows. I don’t think it’ll cause any observable performance difference to just include the whole set. And that way we don’t need to worry about ever needing to update the set in the Wattsi code.

Reply:
> > We definitely use more than these in HTML, but perhaps not in the PR.
> 
> ah, OK, yeah — I was just basing it on what we needed for that PR
> 
> > For example most of the IDL types would make sense. I'd at least like to have attr-value, element-state, constructor, callback, dictionary, dict-member, enum, typedef.
> 
> I guess I can just go ahead and add the complete set of ~40 dfn types that Bikeshed code shows. I don’t think it’ll cause any observable performance difference to just include the whole set. And that way we don’t need to worry about ever needing to update the set in the Wattsi code.

OK, I just went ahead and added the full set of 40. That caused a couple more to be caught that the initial subset of 9 had missed: `attr-value` and `constructor`.